pkg/yamltemplate/filetests/ytt-library/ip/parse-cidr-net-wont-encode.yml

```
@@ -0,0 +1,11 @@
+#@ load("@ytt:ip", "ip")
+
+#@ addr, net = ip.parse_cidr("192.168.0.1/24")
+addr: #@ addr
+
++++
+
+ERR: 
+- __ytt_tplXXX_set_node: Unable to convert value: IPAddrValue does not automatically encode (hint: use .string())
```

```suggestion
- __ytt_tplXXX_set_node: Unable to convert value: @ytt:ip.addr does not automatically encode (hint: use .string())
```

(see rationale in implementation, below)

---

fixed